Documentation/install/upgrade.rst

```diff
@@ -267,6 +267,16 @@ Annotations:
 1.5 Upgrade Notes
 -----------------
 
+Upgrading from >=1.4.0 to 1.5.y
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. The TCP conntrack table size ``ct-global-max-entries-tcp`` from
```

Comment:
Also that the default table size has changed from 1M to 500K and that this change will cause a one-time disruption of all connection on upgrade. In order to avoid this, set the value to 1M.

Reply:
Updated my wording. As discussed IRL, the cilium-agent's default value hasn't changed, so therefore I'm not mentioning about `the default table size has changed <..>`.